cmd/inspect.go

```diff
@@ -211,10 +213,14 @@ func populateManifest(out io.Writer, m *bundle.Manifest) error {
 		lines = append(lines, []string{"Metadata", truncateTableStr(string(metadata))})
 	}
 
-	t.AppendBulk(lines)
-	if t.NumLines() > 0 {
+	if err := t.Bulk(lines); err != nil {
```

Comment:
I suppose we only need to do this if `len(lines) > 0`; so moving this inside the below `if` block could save us a potential error state for that edge case. Unless there is a need to always to this?